documentation/operators/src/changelog.md

```diff
@@ -2,6 +2,84 @@
+[#4573]: https://github.com/nymtech/nym/pull/4573
+[#4564]: https://github.com/nymtech/nym/pull/4564
+[#4553]: https://github.com/nymtech/nym/pull/4553
+[#4552]: https://github.com/nymtech/nym/pull/4552
+[#4548]: https://github.com/nymtech/nym/pull/4548
+~~~
+
+### Features
+
+- New `nym-node` API endpoint `/api/v1/auxiliary-details`: to expose any additional information. Currently it's just the location. `nym-api` will then query all nodes for that information and put it in the `self-described` endpoint.
+- Newly `nym-node` location available - use one of the three options to add this to your node config:
+    1. Update the `location` field under `[host]` section of `config.toml`
+    2. For new nodes: Initialise the node with `--location` flag, where they have to provide the country info. Either full country name (e.g. 'Jamaica'), two-letter alpha2 (e.g. 'JM'), three-letter alpha3 (e.g. 'JAM') or three-digit numeric-3 (e.g. '388') can be provided.
+    3. For existing nodes: It's also possible to use exactly the same `--location` argument as above, but make sure to also provide `--write-changes` (or `-w`) flag to persist those changes!
+- [Feature/unstable tested nodes endpoint](https://github.com/nymtech/nym/pull/4601): Adds new data structures (`TestNode`, `TestRoute`, `PartialTestResult`) to handle test results for Mixnodes and Gateways. With the inclusion of pagination to handle large API responses efficiently. Lastly, introducing a new route with the tag `unstable` thus meaning not to be consumed without a user risk, prefixes in endpoints with unstable, are what it says on the tin.
+    - Testing Steps Performed:
+        1. Deploy new api changes to sandbox environment
+        2. Ensure current operations are transactional and standed operations are working
+        3. Run a script to ensure that the new endpoints are working as expected with pagination 
+
+ <img width="719" alt="image" src="https://github.com/nymtech/nym/assets/60836166/91285971-e82a-4e5a-8a58-880505ae1be9">
+
+- [`nym-api`: make report/avg_uptime endpoints ignore blacklist](https://github.com/nymtech/nym/pull/4599): When querying for node specific data, it's no longer going to go through the entire list of all cached (and filtered nodes) to find it; instead it will attempt to retrieve a single unfiltered entry.
+    - Tests:
+        - Built the project and deployed it in a test environment.
+        - Manually tested API endpoints for mixnode and gateway data.
+        - Verified that the endpoints return the expected data and handle blacklists correctly.
+        - API performance improved due to the efficient `HashMap` lookups
+        - Data in mainnet will differ from test nets due to the increased amount of gateways and mixnodes in that environment
+        - Test standard uptime routes:
+```sh
+curl -X 'GET' 'https://qa-nym-api.qa.nymte.ch/api/v1/status/gateway/3ZmKvV3Fax9A8txxQ3GKh9fUeyTAk68rK4Yn4m3Vdvp9/avg_uptime' -H 'accept: application/json'
```

Comment:
we can use the mainnet data here now for querying this if need be for an example as this has been updated in mainnet
```
curl -X 'GET' 'https://validator.nymtech.net/api/v1/status/gateway/Fo4f4SQLdoyoGkFae5TpVhRVoXCF8UiypLVGtGjujVPf/avg_uptime' -H 'accept: application/json'

Reply:
great! thank you